Nick

nevans224 opened the pull request:
This is the login page and a lot of progress on single recipe plus css

nevans224 committed:
I changed the whole layout and wrote the login.css page. I stopped using modals.css which was probably bad but whatev

nevans224 committed:
Updates to text fields and button

nevans224 committed:
Merge branch 'gh-pages' of https://github.com/FiveGuys-CS256/html-prototype into gh-pages

Conflicts:
	css/modals.css

loberster mac n cheese made. Single recipe css made

nevans224 committed:
Merge branch 'gh-pages' of https://github.com/FiveGuys-CS256/html-prototype into gh-pages

Conflicts:
	css/modals.css

All my work from today so far... Sorry for crappy commit messages

jroweboy commented on css/single-recipe.css:
```diff
@@ -0,0 +1,52 @@
+body {
```
move this to the flex.css or remove it altogether

nevans224 replied:
Done

jroweboy commented:
please remove any css classes in single-recipe.css that is duplicated in login as well.

nevans224 committed:
Deletion of duplicate code in single-recipe.css
Deletion of global variables from single recipe.css
Insert new color onto index.html
